sky/__init__.py

```diff
@@ -104,11 +102,16 @@ def set_proxy_env_var(proxy_var: str, urllib_var: Optional[str]):
 from sky.data import StoreType
 from sky.execution import exec  # pylint: disable=redefined-builtin
 from sky.execution import launch
-from sky.execution import spot_launch
 from sky.optimizer import Optimizer
 from sky.optimizer import OptimizeTarget
 from sky.resources import Resources
 from sky.skylet.job_lib import JobStatus
+# TODO (zhwu): These imports are for backward compatibility, and spot APIs
+# should be called with `sky.spot.xxx` instead. Remove in release 0.7.0
+from sky.spot import cancel as spot_cancel
```

Comment:
To discuss: Shall we add a wrapper and print some warning message when the original API is called? like `sky.spot_cancel is deprecated. It will be removed in release 0.7.0. Please use sky.spot.cancel.`